stingray/pulse/modeling.py

```diff
@@ -0,0 +1,180 @@
+import numpy as np
+from astropy.modeling import models, fitting
+
```

Comment:
I think there should be an `__all__ = ["classes/functions we want accessible from the outside"]` be up here?